### DISCUSSION TITLE
Not separating different emote types in the Emoji window

### DISCUSSION THREAD

**Discussion by ashugg** [Chatterino/chatterino2#5742] [Ideas] [answered]
In the Subs tab of the Emote window, for each subscribed server the emotes are shown grouped together by [type](https://help.twitch.tv/s/article/how-to-use-emotes#get): Follower emotes, Subscriber emotes, Custom Bit Badges and Bit Reward Emotes.

For each type, if the server has any emotes of that type, they are shown together on one or more lines (depending on the window width and the number of emotes). For the next type, any emotes will be shown on a new line, as shown below.

![20241129 Emotes window in Chatterino](https://github.com/user-attachments/assets/3dd76716-91e2-4542-a7d2-9f67ca6c907a)

The Chatterino UI doesn't offer anything to explain why the emotes are grouped in this way, rather than just being shown all together in one block like in the web browser, taking up less space.

![20241129 Emotes window in Browser](https://github.com/user-attachments/assets/8b208e04-c8b5-4d71-add9-caf1d8096069)

(They are also in a different order, but I don't know why. Maybe Chatterino sorts them by ID number and the browser sorts them by emote name?)

I would like to see an option that would make Chatterino follow the browser behaviour. What do other people think of this?

Further options could include: sorting all emotes by name (with and without the server prefix), and sorting by any other criteria. Different sort options in the Emoji tab would be nice, too, as I find it pretty difficult to locate the emoji I want from the list whose sorting I don't understand. (Yes, there's search, but sometimes you can't guess the right word.) All the faces aren't grouped together, for example.

![20241129 Emojis in Chatterino](https://github.com/user-attachments/assets/75d4a8c8-6406-4a8d-912d-13ac5f1797a7)

**Comment by pajlada** [comment, marked as answer]
This should be resolved in the next release/latest nightly. Please help confirm if this is not the case. 

For reference, a screenshot of my emotes. this includes animated emotes, t2/t3 emotes, follower emotes

![image](https://github.com/user-attachments/assets/67cc7bad-1182-4996-a003-bb4524c37b77)

**Comment by ashugg** [comment]
Confirmed resolved in Chatterino 2.5.2 (commit f53d92c7)
